package.json

```diff
@@ -0,0 +1,110 @@
+{
+  "name": "@venusprotocol/protocol-reserve",
+  "description": "",
+  "version": "0.0.1",
+  "author": "",
+  "files": [
```

Comment:
Can we add engines to control the required node version?

Reply:
The README says 12 but I think we should require at least 16. Some of our repos require 16. 18 is the LTS

Reply:
Looks good. Can we update the README too?